lib/src/tab_pane.dart

```diff
@@ -57,7 +57,7 @@ class _TabPaneState extends State<TabPane> {
       final Tab tab = widget.tabs[i];
       if (i == selectedIndex) {
         tabs.add(
-          Ink(
```

Comment:
Does this mean you can get rid of the `material.dart` import?

Reply:
I removed this and also the `rendering.dart` import since I don't believe it's needed any more either.